src/app_service/service/contacts/dto/contacts.nim

```diff
@@ -97,11 +97,36 @@ proc userNameOrAlias*(contact: ContactsDto): string =
   else:
     result = contact.alias
 
-proc isContact*(self: ContactsDto): bool =
-  result = self.added
+proc didHeSendMeContactRequest*(self: ContactsDto): bool =
+  return self.hasAddedUs
+
+proc didISendHimContactRequest*(self: ContactsDto): bool =
+  return self.added
+
+proc didHeRejectMyContactRequest*(self: ContactsDto): bool =
+  # This info will come from the `status-go`
```

Comment:
I'd change this message to `TODO not implemented in status-go yet` since it's clearer
(same for the ones below)